modules/grafana/manifests/dashboards/deployment_dashboard.pp

```diff
@@ -23,6 +23,24 @@
   $app_domain = undef,
   $has_workers = true,
 ) {
+  if $has_workers {
+    $worker_row = [["worker_failures", "worker_successes"]]
+  } else {
+    $worker_row = []
+  }
+
+  $panel_partials = concat(
+    [
+      ["processor_count", "5xx_rate"],
+      ["error_deploy_counts", "recent_500_count", "period_500_count", "links"]
+    ],
+    $worker_row,
+    [
+      ["errors_by_controller_action"],
+      ["response_times_by_controller"]
+    ]
+  )
```

Comment:
`<<` isn't valid Puppet syntax, unfortunately. And you can't use `+` or `+=` to concatenate arrays either.

We've got access to the functions in [puppet stdlib v4.12](https://forge.puppet.com/puppetlabs/stdlib/4.12.0/readme), and `concat` seemed like the least messy option.